collections/_get-started/installation.html

```diff
@@ -1,33 +1,86 @@
 ---
 title: Installation
 layout: layout-sidenav
-description: A dropdown mechanism that allow users to either show or hide related content.
+description: Installation guide that will help you quicky spin up a site using SGDS
 ---
+{% capture reactjs %}
+npm install @govtechsg/sgds-react
+
+//not required if using cdn
+npm install @govtechsg/sgds-govtech bootstrap-icons
+{% endcapture %}
+{% capture using-node %}
+npm i @govtechsg/sgds bootstrap@5.1.3 bootstrap-icons
+{% endcapture %}
+
+{% capture webpack %}
+import "@govtechsg/sgds/css/sgds.css";
+import 'bootstrap';
+// or get all of the named exports for further usage
```

Comment:
Need to import bootstrap? I thought @govtechsg/sgds library already contains bootstrap?

Reply:
oh yes!! great catch!!